sunpy/map/maputils.py

```diff
@@ -410,3 +411,89 @@ def contains_coordinate(smap, coordinates):
             (xc <= xs - point5pix) &
             (yc >= -point5pix) &
             (yc <= ys - point5pix))
+
+
+def _bresenham(x1, y1, x2, y2):
```

Comment:
```suggestion
def _bresenham(*, x1, y1, x2, y2):
```
These should definitely be keyword only, to avoid confusion in order when calling the function.